bucket/qview-nightly.json

```diff
@@ -35,20 +26,21 @@
     ],
     "checkver": {
         "script": [
-            "$api = (Invoke-RestMethod 'https://vsrm.dev.azure.com/jurplel/79795e5b-716b-4a59-a698-8605132587c5/_apis/Release/deployments').value[0].release.artifacts[0].definitionReference.version",
-            "('id', 'name' | ForEach-Object { $api.$_ }) -join ' '"
+            "try {",
+            "    $api = (Invoke-RestMethod 'https://api.github.com/repositories/123522764/actions/workflows/build.yml/runs?branch=master&status=success').workflow_runs[0]",
+            "    ($api.id, $api.updated_at.ToString('yyyy-MM-dd'), $api.run_attempt) -join ' '",
+            "}",
+            "catch { '' }"
```

Comment:
No need to use `try {} catch {}` here

Reply:
it is needed, as if the script fails, it will regex the website instead of the script, it's better for it to regex the empty string, and just fail to upate

Reply:
error message of the script should be shown in the logs, you are trying to hiding them.

Reply:
It isn't really me trying to hide it, the regex shouldn't run if the script throws, but it does currently, I've had many issues with this previously. You could open an issue https://github.com/ScoopInstaller/Scoop but for now, this is the way to deal with it.

Reply:
i dont know what you're facing,

test with a wrong url:
```
❯ bin\checkver qview-nightly
qview-nightly: couldn't match '\A(\d+) (?<date>\d{4}(?:-\d{2}){2}) (?<attempt>\d+)\Z' in https://github.com/jurplel/qView


############## WITHOUT try {} catch {}:
❯ bin\checkver qview-nightly
Invoke-RestMethod: {"message":"Not Found","documentation_url":"https://docs.github.com/rest/actions/workflow-runs#list-workflow-runs-for-a-workflow"}
InvalidOperation:
Line |
   2 |  ($api.id, $api.updated_at.ToString('yyyy-MM-dd'), $api.run_attempt) - …
     |  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | You cannot call a method on a null-valued expression.
MethodInvocationException: C:\Users\humorce\scoop\apps\scoop\current\bin\checkver.ps1:348
Line |
 348 |  …               $match = $regex.Matches($page) | Select-Object -First 1
     |                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Exception calling "Matches" with "1" argument(s): "Value cannot be null. (Parameter 'input')"

qview-nightly: couldn't match '\A(\d+) (?<date>\d{4}(?:-\d{2}){2}) (?<attempt>\d+)\Z' in https://github.com/jurplel/qView
```

Reply:
I don't remember if I could reproduce it locally, it was more of an issue with excavator for me

Reply:
I have not meet this issue before, tracked here: https://github.com/ScoopInstaller/GithubActions/issues/36